Tests/DoublyLnkList.cs

```diff
@@ -1,61 +1,219 @@
+            if (_last == null)
+            {
+                _head = newNode;
+                _last = newNode;
+            }
+            else
+            {
+                newNode.Previous = _last;
+                _last.Next = newNode;
+                _last = newNode;
+            }
+            _count++;
+        }
 
-    public int Count()
-    {
-        throw new NotImplementedException();
-    }
+        public T First()
+        {
+            if (_head == null)
+                throw new InvalidOperationException();
+            return _head.Value;
+        }
 
-    public IEnumerable<T> ToEnumerable()
-    {
-        throw new NotImplementedException();
-    }
+        public bool Any(Func<T, bool> compare)
+        {
+            var current = _head;
+            while (current != null)
+            {
+                if (compare(current.Value))
+                    return true;
+                current = current.Next;
+            }
+            return false;
+        }
 
-    public IEnumerable<int> ToReversedEnumerable()
-    {
-        throw new NotImplementedException();
-    }
+        public T Get(int index)
+        {
+            if (index < 0 || index >= _count)
+            {
+                throw new IndexOutOfRangeException();
+            }
 
-    public bool RemoveAt(int index)
-    {
-        throw new NotImplementedException();
-    }
+            var current = _head;
+            for (int i = 0; i < index; i++)
+            {
+                if (current == null)
+                {
+                    throw new InvalidOperationException();
+                }
+                current = current.Next;
+            }
 
-    public bool Remove(T value)
-    {
-        throw new NotImplementedException();
+            if (current == null)
+            {
+                throw new InvalidOperationException();
+            }
+
+            return current.Value;
+        }
+
+        public int Count()
+        {
+            return _count;
+        }
+
+        public IEnumerable<T> ToEnumerable()
+        {
+            var current = _head;
+            while (current != null)
+            {
+                yield return current.Value;
+                current = current.Next;
+            }
+        }
+
+        public IEnumerable<T> ToReversedEnumerable()
+        {
+            var current = _last;
+            while (current != null)
+            {
+                yield return current.Value;
+                current = current.Previous;
+            }
+        }
+
+        public bool RemoveAt(int index)
+        {
+            if (index < 0 || index >= _count)
+                return false;
+
+            var current = _head;
+            for (int i = 0; i < index; i++)
+            {
+                if (current == null)
+                {
+                    throw new InvalidOperationException();
+                }
+                current = current.Next;
+            }
```

Comment:
Esto me parece muy extraño, por que iteras hasta el final? Aunque está buena la idea de usar un for cuando necesitas llevar conteo del indice.